### PR TITLE
✨ Key Features | Homepage Carousel and Cleanup

### PR DESCRIPTION
**Pull request by joshbermanssw**
This is a component I had built a while ago but was never finished due to needing media for it. However because it is so old and we have updated some of its dependent components/ methods i decided it'd be easier to re-build it. 

PBI https://github.com/tinacms/tina.io/issues/1805

Old PR https://github.com/tinacms/tina.io/pull/1816

This is a component built from @bettybondoc's original re-design of the homepage from figma: https://www.figma.com/design/Jyj0SszjfSBc6Minkl3g1l/Tina.io?node-id=399-5400&node-type=FRAME

**Changes:**
- added another button variant, ghost (blue) which is the same as ghost normal but blue text instead of red
- created new CarouselFeature component and template
- removed 2 old feature blocks from the homepage (as per figma design) 

**Still todo**
- [x] media rendering with using the `image` field rather than src

### CODE REVIEW

**Review by isaaclombardssw — CHANGES_REQUESTED** [review]
Sick component 🚀 

Only thing that has to change is the below (missing content in 1st and 3rd RHS option).

File: components/blocks/CarouselFeature.tsx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+        intervalRef.current = setInterval(() => {
+          setHoveredIndex((prevIndex) => {
+            if (prevIndex === null || prevIndex === data?.items?.length - 1) {
+              return 0;
+            }
+            return prevIndex + 1;
+          });
+        }, 3000);
+      }
+    };
```
Comment (isaaclombardssw) [inline components/blocks/CarouselFeature.tsx:156]: Could do with some comments.

**Review by isaaclombardssw — APPROVED** [review]
Code looks aight 👍

File: components/blocks/CarouselFeature.tsx [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+  FaFileAlt,
+  AiOutlineUser,
+  BiBadge,
+  BiSupport,
+  AiOutlineUsergroupAdd,
+  CgCrown,
+  HiOutlineSparkles,
+  TbPlugConnected,
+  SlLock,
+};
```
[the remark this answers is not in the document]
Comment (joshbermanssw) [reply components/blocks/CarouselFeature.tsx]: done

File: components/blocks/CarouselFeature.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/blocks/CarouselFeature.tsx:153]: done

**Pull request merged**